File: KingdomCodeTool/wwwroot/Download/202501201119/Angular/shared/T_FK_Xref.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { T_FK_Xref } from './T_FK_Xref.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class T_FK_XrefService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'ID', 'ConstraintName', 'MasterTable', 'MasterColumn', 'ChildTable', 'ChildColumn', 'FKOrder'];

    List: T_FK_Xref[] | undefined;
    ListFilter: T_FK_Xref[] | undefined;
    FormData!: T_FK_Xref;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "T_FK_Xref";
    }
}

File: KingdomCodeTool/wwwroot/Download/202501201119/Service/Implement/T_FK_XrefService.cs
namespace Service_eHospital_DongNai_A_System.Implement
{
    public class T_FK_XrefService : BaseService<T_FK_Xref, IT_FK_XrefRepository>
    , IT_FK_XrefService
    {
    private readonly IT_FK_XrefRepository _T_FK_XrefRepository;
    public T_FK_XrefService(IT_FK_XrefRepository T_FK_XrefRepository) : base(T_FK_XrefRepository)
    {
    _T_FK_XrefRepository = T_FK_XrefRepository;
    }
    }
    }

